main.py

```diff
@@ -18,24 +19,25 @@ def signup():
     password = request.form["password"]
     verify = request.form["verify"]
     email = request.form["email"]
-    string_len = True
+    
+    username_error = ""
+    password_error = ""
+    verify_error = ""
+    email_error = ""
+    
     if username == "" or check_character_len(username) == False:
         username_error = "That's not a valid username"
-    else:
-        username_error = ""
+
     if password == "" or check_character_len(password) == False:
         password_error = "That's not a valid password"
```

Comment:
In general the username and password errors are not super helpful, no where do we specify _what_ qualifies as a valid password/username.

Reply:
agree on this one, was just copying the finished versions messages